### PR TITLE
FEAT: Michigan scraper

### PR DESCRIPTION
**Pull request by tlyon3**

### CODE REVIEW

File: src/cmdc_tools/datasets/official/MI/data.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    def get(self):
+        urls = self._get_urls()
+        cases = self._get_cases(urls[0])
+
+        return cases
+
+    def _get_urls(self):
+        loop = asyncio.new_event_loop()
+        asyncio.set_event_loop(loop)
+        urls = loop.run_until_complete(self._get_urls_async())
```
Comment (sglyon) [inline src/cmdc_tools/datasets/official/MI/data.py]: We are running on python 3.7+

Let's just do `urls = asyncio.run(self._get_urls_async())`

In python 3.7 the internal Python team exposed that function and now they handle setting up/closing down the event loop ([ref](https://docs.python.org/3/library/asyncio-task.html#asyncio.run))

File: src/cmdc_tools/datasets/official/MI/data.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        cases_url_raw = await page.Jx("//a[text()='Cases by County by Date']/..")
+        cases_url = await cases_url_raw[0].Jeval("a", "node => node.href")
+        # tests_url_raw = await page.Jx("//a[text()='COVID-19 Tests by County']/..")
+        # tests_url = await tests_url_raw[0].Jeval("a", "node => node.href")
+
+        # return [cases_url, tests_url]
+        return [cases_url]
+
+    def _get_cases(self, url):
+        df = pd.read_excel(url)
```
Comment (cc7768) [inline src/cmdc_tools/datasets/official/MI/data.py]: Is this successfully parsing `Date` as a `datetime`? It might make sense to add `parse_dates=["Date"]` to the request.

File: src/cmdc_tools/datasets/official/MI/data.py [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+    def _get_urls(self):
+        urls = asyncio.run(self._get_urls_async())
+        return urls
+
+    async def _get_urls_async(self):
+        browser = await pyppeteer.launch()
+        page = await browser.newPage()
+        await page.goto(
+            "https://www.michigan.gov/coronavirus/0,9753,7-406-98163_98173---,00.html"
```
Comment (cc7768) [inline src/cmdc_tools/datasets/official/MI/data.py]: It looks like we can get more than just cases and deaths from this page. They also have a tab for testing which has positive/negative tests by date and county.

**Review by cc7768 — CHANGES_REQUESTED** [review]

File: src/cmdc_tools/datasets/official/MI/data.py (hunk 2 above)
Comment (tlyon3) [reply src/cmdc_tools/datasets/official/MI/data.py]: It's parsing it. `df.info` shows the `dt` column as a `datetime64`

File: src/cmdc_tools/datasets/official/MI/data.py (hunk 3 above)
Comment (tlyon3) [reply src/cmdc_tools/datasets/official/MI/data.py]: Getting it now

File: src/cmdc_tools/datasets/official/MI/data.py (hunk 1 above)
Comment (tlyon3) [reply src/cmdc_tools/datasets/official/MI/data.py]: Updated
